src/main/java/com/softserve/demo/filter/ProviderFilter.java

```diff
@@ -3,40 +3,39 @@
 import com.softserve.demo.dto.ProviderInfoDTO;
 import com.softserve.demo.filter.specification.ProviderSpecification;
 import com.softserve.demo.model.Provider;
+import com.softserve.demo.model.Service;
 import com.softserve.demo.model.search.ProviderCriteria;
 import com.softserve.demo.service.impl.ProvidersServiceImpl;
 import com.softserve.demo.service.impl.ServiceFromProvidersImpl;
+import com.softserve.demo.util.Constant;
 import com.softserve.demo.util.mappers.ProviderInfoMapper;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageImpl;
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.jpa.domain.Specification;
-import org.springframework.stereotype.Service;
-
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
-@Service
+@org.springframework.stereotype.Service
```

Comment:
Why to import in this way

Reply:
Because I have another Service from model package

Reply:
Oh, no problem